README.rst

```diff
@@ -78,9 +78,34 @@ supports both installing your ``project.dependencies`` as well as your
 ``project.optional-dependencies``. Thanks to the fact that this is an
 official standard, you can use ``pip-compile`` to pin the dependencies
 in projects that use modern standards-adhering packaging tools like
-`Hatch <https://hatch.pypa.io/>`_ or `flit <https://flit.pypa.io/>`_.
+`Setuptools <https://setuptools.pypa.io>`_ , `Hatch <https://hatch.pypa.io/>`_
+or `flit <https://flit.pypa.io/>`_.
 
-Suppose you have a Django application that is packaged using ``Hatch``, and you
+Suppose you have a 'foobar' Python application that is packaged using ``Setuptools``,
+and you want to pin it for production. You can declare the project metadata as:
+
+.. code-block:: toml
+
+    [build-system]
+    requires = ["setuptools", "setuptools-scm"]
+    build-backend = "setuptools.build_meta"
+
+    [project]
+    name = "foobar"
+    description = "foobar description"
+    classifiers = [
+        "Programming Language :: Python",    
+        "Programming Language :: Python :: 3",
+    ]
+    dynamic = ["version", "dependencies"]
+
+    [project.urls]
+    homepage = "https://github.com/xyz/foobar"
```

Comment:
Also remove project.urls section because is unrelated to our example. It is correct and recommended for setuptools but outside the scope of our example.